tasks/message/message.go

```diff
@@ -266,7 +268,20 @@ func (p *MessageProcessor) extractMessageModels(ctx context.Context, ts *types.T
 				Nonce:      message.Nonce,
 				Method:     uint64(message.Method),
 				Params:     message.Params,
```

Comment:
Can we drop the Params column from Messages now we have ParsedMessages? Seems wasteful to store the data twice and it would make the messages row width a lot smaller.

Reply:
I'm okay with that. the only downside is that it's destructive. if we have messages we've seen on previous forks that a back-fill won't re-find, we shouldn't delete that data.